cmake/configure_files/wcs_config.hpp.in

```diff
@@ -5,6 +5,22 @@
 #define WCS_VERSION @WCS_VERSION@
 #cmakedefine WCS_GIT_VERSION @WCS_GIT_VERSION@
 
+/* Defined for the generation of the c++ code of a sbml file */
+#cmakedefine CMAKE_CXX_COMPILER "@CMAKE_CXX_COMPILER@"
```

Comment:
Let's try something here.
Instead of, for example,
```
#cmakedefine CMAKE_CXX_FLAGS "@CMAKE_CXX_FLAGS@"
#ifndef CMAKE_CXX_FLAGS
#define CMAKE_CXX_FLAGS ""
#endif
```
```
#define CMAKE_CXX_FLAGS "@CMAKE_CXX_FLAGS@"
```